addons/packages/fluxcd-helm-controller/0.19.0/package.yaml

```
@@ -0,0 +1,38 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: helm-controller.fluxcd.community.tanzu.vmware.com:0.19.0
```

```suggestion
  name: helm-controller.fluxcd.community.tanzu.vmware.com.0.19.0
```

@hemakshis The name of the package needs to be of the format `<spec.refName> + '.' + <spec.version>`

---

fixed

---

apologies, in the previous commit this fix was not pushed but now i have fixed it.